packages/tools/src/tools/PanTool.ts

```diff
@@ -29,6 +29,12 @@ export default class PanTool extends BaseTool {
     const enabledElement = getEnabledElement(element);
 
     const deltaPointsWorld = deltaPoints.world;
+    if (
+      deltaPointsWorld[0] === 0 &&
+      deltaPointsWorld[1] === 0 &&
+      deltaPointsWorld[2] === 0
+    )
+      return;
```

Comment:
The linter should warn you here (if not needs to be fixed), throughout the repo we always use curly brackets for the if statements